src/AaveV3StrategicAssets_20230606/interfaces/IWardenBoost.sol

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import {IERC20} from 'solidity-utils/contracts/oz-common/interfaces/IERC20.sol';
+
+interface IWardenBoost {
+  struct BoostOffer {
+    // Address of the user making the offer
+    address user;
+    // Price per vote per second, set by the user
+    uint256 pricePerVote;
+    // Max duration a Boost from this offer can last
+    uint64 maxDuration;
+    // Timestamp of expiry of the Offer
+    uint64 expiryTime;
+    // Minimum percent of users voting token balance to buy for a Boost
+    uint16 minPerc; //bps
+    // Maximum percent of users total voting token balance available to delegate
+    uint16 maxPerc; //bps
+    // Use the advised price instead of the Offer one
+    bool useAdvicePrice;
+  }
+
+  /// @notice Claim rewards for selling boost
+  function claim() external returns (bool);
+
+  /// @notice Retrieve the veBoost address for Warden contract
+  function delegationBoost() external returns (address);
+
+  /// @param boostId Id of Boost to check offers for
+  function offers(uint256 boostId) external returns (BoostOffer memory);
+
+  /**
```

Comment:
This NatSpec is using a different multi-line comment pattern than the rest-- recommend changing just for aesthetics